lessons/javascript-fundamentals/functions-deep-dive/README.md

```diff
@@ -0,0 +1,61 @@
+Title | Tags
+--- | ---
+JavaScript Fundamentals: Workshop Review | javascript, arrays, conditionals, loops, functions
+
+## Objectives
+* Learn how arrays work under the hood
+* Be able to confidently answer all wkshp assessment questions (ex: find second largest element in an input array)
```

Comment:
Good call! I was looking for a good place to talk about learning how to learn, and this should be a great spot